lib/Reply.js

```diff
@@ -84,6 +90,9 @@ class Reply {
     this.msg.reprompt = reply.msg.reprompt || this.msg.reprompt;
     this.msg.card = reply.msg.card || this.msg.card;
     this.msg.directives = reply.msg.directives || this.msg.directives;
+    if (reply.msg.directives) {
+      this.msg.directives = _.concat(this.msg.directives, reply.msg.directives);
```

Comment:
Seems like this combined with the line above it could have some unintended consequences. I'd recommend just dropping the above line and the if statement?